doc/development.md

```diff
@@ -50,6 +50,16 @@ docker run --rm -e TARGET=amd64 -e ACTION=COMPILE -v "$PWD":/usr/src/myapp -w /u
 sudo -E PATH=$PWD/bin/amd64:$PATH ./tests.test
 ```
 
+## Runnning Blackbox Tests locally
```

Comment:
> Also do you think we should list the programs which'd in set_ld_flags_host.sh, or is the package list under Test Host System Dependencies sufficient?

I'll add `mdadm`, otherwise it's sufficient. With the exception of uuid-runtime (which is only sometimes needed), they're all the names of the upstream packages.

Reply:
My concern is that package names can vary between distros, by "upstream" do you mean Gentoo?

Reply:
No, I mean the names of the actual projects themselves.

Reply:
Oh yeah that's fine then.